nunaserver/nunaserver/generator.py

```diff
@@ -169,24 +171,53 @@ def generate_dsdl(
         generator.generate_all()
         support_generator.generate_all()
 
-    # Zip result
-    zipfile_name = f"nunavut_out-{uuid.uuid4()}.zip"
-    zipf = zipfile.ZipFile(f"/tmp/{zipfile_name}", "w", zipfile.ZIP_DEFLATED)
-    zipdir(out_dir, zipf)
-    zipf.close()
-
-    # Upload result
-    storage.fput_object(
-        f"results",
-        zipfile_name,
-        f"/tmp/{zipfile_name}",
-        os.stat(f"/tmp/{zipfile_name}").st_size,
-    )
+    if target_lang == "html":
+        # Upload generated files
+        for file in out_dir.glob("**/*"):
+            if file.is_file():
+                rel_path = doc_url / file.relative_to(out_dir)
+                try:
+                    if storage.stat_object("docs", f"{rel_path}"):
+                        raise RuntimeError("Specified doc URL is already taken")
+                except minio.error.S3Error:
```

Comment:
Some logging is in order I suppose

Reply:
That's something we have to decide. I don't have a delete policy right now; I can set it to 30 days, same as how long it keeps results.